cmd/experimental/kjobctl/pkg/builder/builder.go

```diff
@@ -478,7 +478,6 @@ func (b *Builder) Do(ctx context.Context) (runtime.Object, []runtime.Object, err
 	if err := b.setClients(); err != nil {
 		return nil, nil, err
 	}
-
```

Comment:
Please revert it.

Reply:
i didn't get what to revert  theirs  not any black space .

Reply:
Looks like you deleted this line. It's not necessary for this PR.